src/Jobs_Templete/RegularJob.yaml.template

```diff
@@ -110,6 +112,12 @@ spec:
       valueFrom:
         fieldRef:
           fieldPath: status.podIP
+    - name: DLWS_GID
+      value: "{{ job["gid"] }}"
+    - name: DLWS_UID
+      value: "{{ job["uid"] }}"
+    - name: DLWS_USER
```

Comment:
Please use `DLWS_USER_NAME`. I have added this env in my [PR](https://github.com/microsoft/DLWorkspace/pull/254/files?file-filters%5B%5D=.json&file-filters%5B%5D=.py&file-filters%5B%5D=.template&file-filters%5B%5D=.yaml&file-filters%5B%5D=No+extension&file-filters%5B%5D=dotfile#diff-8fe18c38c935607319ae7161b2038889R82).

Reply:
Cool, will update